diff -urN a-3.3.0/Include/pyport.h b-3.3.0/Include/pyport.h
--- a-3.3.0/Include/pyport.h	Fri Nov 16 17:28:36 2012
+++ b-3.3.0/Include/pyport.h	Fri Nov 16 17:30:39 2012
@@ -241,6 +243,8 @@
 #       define PY_FORMAT_SIZE_T "l"
 #   elif defined(MS_WINDOWS)
 #       define PY_FORMAT_SIZE_T "I"
+#   elif defined(__MINGW32__) && defined(__USE_MINGW_ANSI_STDIO)
+#       define PY_FORMAT_SIZE_T "z"
 #   else
 #       error "This platform's pyconfig.h needs to define PY_FORMAT_SIZE_T"
 #   endif
